src/main/java/org/folio/circulation/domain/UpdateRequestQueue.java

```diff
@@ -184,8 +185,15 @@ private RequestQueue setHoldShelfExpirationDateWithExpirationDateManagement(
 
     ExpirationDateManagement expirationDateManagement = calculatedRequest.getPickupServicePoint()
       .getHoldShelfClosedLibraryDateManagement();
-    String intervalId = calculatedRequest.getPickupServicePoint().getHoldShelfExpiryPeriod()
-      .getIntervalId().toUpperCase();
+
+    String intervalId = "0";
```

Comment:
Can you use another value instead of this magic number?

Reply:
updated